src/EnergyPlus/RoomAirModelAirflowNetwork.cc

```diff
@@ -83,6 +85,7 @@
 #include <EnergyPlus/RoomAirModelAirflowNetwork.hh>
 #include <EnergyPlus/SteamBaseboardRadiator.hh>
 #include <EnergyPlus/UtilityRoutines.hh>
+#include <EnergyPlus/ZoneAirLoopEquipmentManager.hh>
```

Comment:
Including this won't cause a circular dependency, right?  I don't want red squiggles to start showing up for @rraustad again.  @rraustad , perhaps you could open this branch in visual studio and make sure the changed and affected files here look fine to you.